src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -12,6 +12,7 @@
 /**
  * @ownership Seo
  */
+@Test(groups = {"Seo", "SeoHtmlTitle"})
 public class HtmlTitleTests extends NewTestTemplate {
 
   // TODO: QAART-681 Sonar should not report duplicate strings warnings for data providers
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1135)